src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -81,6 +81,11 @@
           text: az storage container create -n MyStorageContainer --fail-on-exist
 """
 
+helps['storage account create'] = """
```

Comment:
So what happens if I do have API version specific help?  Does the help file break down?